File: NetFramework/TouchToolkit/VB/TouchToolkitDemo/DemoBase.vb
Imports System.Windows.Forms

Partial Public Class DemoBase
    Inherits UserControl
    Public Sub New()
        InitializeComponent()

        Me.Title = Me.[GetType]().Name
    End Sub

    Public Property Title() As String
        Get
            Return Me.lblTitle.Text
        End Get
        Set(value As String)
            Me.lblTitle.Text = Value
        End Set
    End Property

    Public Property Description() As String
        Get
            Return Me.lblDescription.Text
        End Get
        Set(value As String)
            Me.lblDescription.Text = Value
        End Set
    End Property
End Class

File: Core/TouchToolkit/CS/TouchToolkitDemo/DemoBase.cs
using System.Windows.Forms;

namespace TouchToolkitDemo
{
    public partial class DemoBase : UserControl
    {
        public DemoBase()
        {
            InitializeComponent();
            Title = GetType().Name;
        }

        public string Title
        {
            get => lblTitle.Text;
            set => lblTitle.Text = value;
        }

        public string Description
        {
            get => lblDescription.Text;
            set => lblDescription.Text = value;
        }
    }
}
